cmd/export.go

```diff
@@ -26,7 +26,7 @@ import (
 
 var Exporter *ctl.ExportCommand
 
-func newExportCommand(_ io.Reader, _, _ io.Writer) *cobra.Command {
+func newExportCommand(stdin io.Reader, stdout, stderr io.Writer) *cobra.Command {
```

Comment:
ah - this one isn't actually using the passed in vals which is breaking the linter

Reply:
Oops, missed that. Updated.